packages/namekit-react/src/components/Input.tsx

```diff
@@ -55,11 +55,11 @@ export const Input: React.FC<InputProps> & { Slot: React.FC<SlotProps> } = ({
   const variantClasses = {
```

Comment:
When an `Input` is disabled, the borders shouldn't animate on hover.

Reply:
Done